hashicorp/boundary-ui
# refactor: 💡 Manage dropdown HDS refactoring for managed-groups

Updated the manage dropdown in the managed groups section to reflect HDS design

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14603

## Screenshots (if appropriate)
**Before**
<img width="1799" alt="mangroups before" src="https://github.com/user-attachments/assets/732447bb-ea6c-48d3-a424-db2d46d86d22">
<img width="1799" alt="mangroups before2" src="https://github.com/user-attachments/assets/70a440c3-6292-4003-ad78-9353cbe5023a">

**After**
<img width="1799" alt="mangroups after1" src="https://github.com/user-attachments/assets/fdf93079-ca9f-4ffb-b06f-f0c440655230">
<img width="1799" alt="mangroups after2" src="https://github.com/user-attachments/assets/413277b6-985a-429e-a572-0b40c1a93987">


## How to Test
1. Start up server/Boundary client
2. Navigate to Auth Methods on sidebar
3. Select a resource with type _**LIDC**_ or _**OIDC**_
4. Select managed group from sidebar and select a resource
5. HDS Design should be reflected. 

## Checklist


~- [ ] I have added JSON response output for API changes~
~- [ ] I have added steps to reproduce and test for bug fixes in the description~
~- [ ] I have commented on my code, particularly in hard-to-understand areas~
~- [ ] My changes generate no new warnings~
~- [ ] I have added tests that prove my fix is effective or that my feature works~
